locales/es/LC_MESSAGES/django.po

```diff
@@ -193,3 +189,6 @@ msgstr "%(areacode)s no es un prefijo telefónico válido."
 #: project/util/phone_number.py:70
 msgid "This does not look like a U.S. phone number. Please include the area code, e.g. (555) 123-4567."
 msgstr "Ese no parece un número de teléfono de los Estados Unidos de América. Por favor, incluye el prefijo telefónico, por ejemplo, (555) 123-4567."
+
+#~ msgid "This tool has been suspended! Please reload the page for more details."
```

Comment:
Is this autogenerated? since we don't use these strings anywhere anymore it seems weird to leave them commented out

Reply:
yes this is auto-gen! Not totally sure why the commented out